app/controllers/grade_entry_forms_controller.rb

```diff
@@ -256,6 +257,8 @@ def get_mark_columns
       {
         id: column.id,
         content: column.name + ' (' + column.out_of.to_s + ')',
+        sortable: true,
+        compare: "compare_gradebox"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.